plugin/providers/keboola/provider.go

```diff
@@ -20,21 +20,23 @@ func Provider() terraform.ResourceProvider {
 		},
 
 		ResourcesMap: map[string]*schema.Resource{
-			"keboola_storage_table":            resourceKeboolaStorageTable(),
-			"keboola_storage_bucket":           resourceKeboolaStorageBucket(),
-			"keboola_transformation":           resourceKeboolaTransformation(),
-			"keboola_transformation_bucket":    resourceKeboolaTransformationBucket(),
-			"keboola_gooddata_writer":          resourceKeboolaGoodDataWriter(),
-			"keboola_gooddata_writer_table":    resourceKeboolaGoodDataTable(),
-			"keboola_gooddata_user_management": resourceKeboolaGoodDataUserManagement(),
-			"keboola_snowflake_writer":         resourceKeboolaSnowflakeWriter(),
-			"keboola_snowflake_writer_tables":  resourceKeboolaSnowflakeWriterTables(),
-			"keboola_postgresql_writer":        resourceKeboolaPostgreSQLWriter(),
-			"keboola_postgresql_writer_tables": resourceKeboolaPostgreSQLWriterTables(),
-			"keboola_access_token":             resourceKeboolaAccessToken(),
-			"keboola_orchestration":            resourceKeboolaOrchestration(),
-			"keboola_orchestration_tasks":      resourceKeboolaOrchestrationTasks(),
-			"keboola_csvimport_extractor":      resourceKeboolaCSVImportExtractor(),
+			"keboola_storage_table":               resourceKeboolaStorageTable(),
+			"keboola_storage_bucket":              resourceKeboolaStorageBucket(),
+			"keboola_transformation":              resourceKeboolaTransformation(),
+			"keboola_transformation_bucket":       resourceKeboolaTransformationBucket(),
+			"keboola_gooddata_writer":             resourceKeboolaGoodDataWriter(),
+			"keboola_gooddata_writer_v3":          resourceKeboolaGoodDataWriterV3(),
+			"keboola_gooddata_writer_table":       resourceKeboolaGoodDataTable(),
+			"keboola_gooddata_user_management":    resourceKeboolaGoodDataUserManagement(),
+			"keboola_gooddata_user_management_v2": resourceKeboolaGoodDataUserManagementAppV2(),
```

Comment:
Probably remove the `App` from the function name, so that it matches the previous/legacy one, only differing by the introduced `V2`.